docs/can-stache-element.md

```diff
@@ -206,6 +206,31 @@ document.body.querySelector("button#remove").addEventListener("click", () => {
 @codepen
 @highlight 14-23,only
 
+
+#### Observable class fields
+
+`StachElement` class fields are observables like [can-stache-element/static.props static props],
+ the counter example can be like the following:
+
+ ```html
+ <count-er></count-er>
+<script type="module">
+import { StacheElement } from "can";
+class Counter extends StacheElement {
+  static view = `
+    Count: <span>{{ this.count }}</span>
+    <button on:click="this.increment()">+1</button>
+  `;
+  
+  count: 6
+}
+customElements.define("count-er", Counter);
+</script>
```

Comment:
Can you also add a note in that you must either add the element to the page or call `initialize` in order for the `prop` value to be observable?

Reply:
Done!